docusaurus/docs/Flutter/05-advanced/08-custom-data.mdx

```diff
@@ -0,0 +1,69 @@
+---
+title: Custom Data
+slug: /custom-data
+sidebar_position: 8
+description: Learn how to add and read custom data in the Stream Video Flutter SDK.
+---
+
+Custom data is additional information that can be added to the default data of Stream. It is a dictionary of key-value pairs that can be attached to users, events, and pretty much almost every domain model in the Stream SDK.
+
+On Flutter, custom data is represented by the `Map<String, Object>`. This means that the key must be a string and the value can be any object.
+
+## Adding Custom Data
+
+Adding extra data can be done through the Server-Side SDKs or through the Client SDKs. In the Flutter Stream Video SDK, you can add extra data when creating/updating a call, updating a user and sending event or reaction.
+
+```dart
+// Example of updating the call custom data.
+call.update(custom: {'mycustomfield': 'mycustomvalue'});
+
+// Example of sending a reaction with custom data.
+call.sendReaction(
```

Comment:
Can we separate out custom data addition in calls vs reactions vs whatever in sub headings for SEO reasons? People can't find it with a google search if we mix them all together